bindings/py/cpp_src/plugin/PyBindRegion.cpp

```diff
@@ -587,7 +587,7 @@ namespace nupic
 										}
 
                     // make regionLevel optional and default to false.
```

Comment:
default to true ?

Reply:
yes, that is what they had before.  I think they were trying to make the simplest case require fewer spec parameters where there is only one input or one output.

Actually it does not matter because all of the fields are normally specified in the Spec anyway.

Reply:
I was just pointing out the comment does not match the code, 
> default to false

Reply:
> I was just pointing out the comment does not match the code,

Oh, I missed that...I will fix the comment on the next PR.  Thanks